src/widgets/gauge.rs

```diff
@@ -375,4 +363,25 @@ mod tests {
                 .remove_modifier(Modifier::DIM)
         )
     }
+
+    #[test]
+    fn line_gauge_default_values() {
+        // TODO: replace to `assert_eq!(LineGauge::default(), LineGauge::default())`
+        // when `Eq` or `PartialEq` is implemented for `LineGauge`.
```

Comment:
Is there a blocker to implementing these (or is it just that doing so means a bunch of changes to all the types?)

Reply:
```rust
#[derive(Debug, Default, PartialEq)] // <-- if add `PartialEq`
pub struct LineGauge<'a> {
    block: Option<Block<'a>>,
    ratio: f64,
    label: Option<Line<'a>>,
    line_set: symbols::line::Set, // <-- we'll need this to have PartialEq trait
    style: Style,
    gauge_style: Style,
}
```

So, I suppose deferring it would be nice.